scripts/build.mts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env zx
+import 'zx/globals';
+import { cliArguments, workingDirectory } from './setup/shared.mts';
+
+const [folder, ...args] = cliArguments();
+
+const buildArgs = [...args, '--', '--all-targets', '--all-features'];
```

Comment:
I'm surprised to see `--all-targets` here -- I tried building a couple of programs with `cargo build-sbf -- --all-targets` and it fails because some dependencies that are incompatible with the chain are probably brought in.

Is that what you're trying to test here? It's pretty cool!

Reply:
Yeah, it is a sanity check.